README.adoc

```diff
@@ -1,10 +1,24 @@
-= CloudBees action: Deploy to Kubernetes cluster using a Kustomize config
+= CloudBees action: Deploy to a {K8S} cluster using a Kustomize configuration
 
-Use this action to customize Kubernetes resources by applying overlays or changes to base YAML manifests without directly modifying them.
+Use this action to customize {K8S} configurations and resources with the tool Kustomize, which uses overlays to change base YAML manifests without directly modifying them.
 
 == Prerequisites
 
-Make sure to add link:https://github.com/cloudbees-io/configure-aws-credentials[CloudBees action: Configure AWS credentials] to your YAML file, to initialize the connection with your AWS credentials.
+Make sure to initialize the connection using your AWS credentials, by adding the following to your YAML file:
+
+[source,yaml]
+----
+      - name: Check out repo
```

Comment:
Since "Check out repo" step is not required to initialise the connection using aws credentials. Instead of it ,add below step:

        name: Configure AWS credentials
        uses: cloudbees-io/configure-aws-credentials@v1
        with:
          aws-access-key-id: ${{ secrets.AUTOMATED_TEST_KUBECTL_ACCESS_KEY }}
          aws-secret-access-key: ${{ secrets.AUTOMATED_TEST_KUBECTL_SECRET_KEY }}
          aws-region: aws-region-1